Gemfile

```diff
@@ -55,6 +55,7 @@ group :system_tests do
   gem "puppet-module-posix-system-r#{minor_version}",                            :require => false, :platforms => "ruby"
   gem "puppet-module-win-system-r#{minor_version}",                              :require => false, :platforms => ["mswin", "mingw", "x64_mingw"]
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '>= 3')
+  gem 'beaker-puppet', '~>1.0'
```

Comment:
Wouldn't mind a space for consistency, but 🤷‍♂️

Reply:
@MikaelSmith I can add that space. Was still working on trying to test this.